.vscode/launch.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "name": "Launch Package",
+        "type": "go",
+        "request": "launch",
+        "mode": "debug",
+        "program": "${workspaceFolder}/cmd/"
+      }
+    ]
+  }
```

Comment:
Is this file typically committed to repo? And if so, please add newline at the end of the file.

Reply:
No, .gitignore needs to have this file added

Reply:
fixed

Reply:
File still present. Needs to be deleted (ignored just affects the future).